.gitignore

```diff
@@ -25,6 +25,8 @@ coverage/*
 # Generated Schema used in some tooling. Versioned Schema is uploaded as a Release artifact to Github.
 schema.graphql
 
+.phpcs/*.json
```

Comment:
Do we really need this folder in the repo? 
I don't remember if PHPCS requires the folder to already exist, but if it does, maybe we could just recycle `tests\_output\`.